GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
+  - Approve grant applications
+  - Enter into contracts on behalf of the project 
+  - Oversee contract and legal engagement for the project
+- Participate in at least one call per quarter;
+- Can be part of the Admin team for the project on GitHub
+- Publish minutes of their meeting on [the forum](https://forum.openrefine.org/tag/minute-advisory)
+
+**How to join the Advisory Committee?** Members of the Advisory Committee publish an announcement on the blog when searching for a new member. The Advisory Committee selects new committee members through consensus-driven discussions among current members.
+
+**Conflict of Interest**: In the event of any conflict of interest (a Committee Member, their family member, or someone with whom the Committee Member has a close academic or employment relationship is involved in a decision), the Committee Member must immediately notify other Committee Members. The Committee Members will be asked to recuse themselves from ongoing conversations and decision processes regarding the Transaction.
+
+**Current list of Advisory Committee members**
+- Jan Ainali [@Ainali](https://github.com/Ainali) and on the [forum](https://forum.openrefine.org/u/ainali/summary)
+- Julie Faure-Lacroix [@jfaurelacroix](https://github.com/jfaurelacroix) and on the [forum](https://forum.openrefine.org/u/jfaurel/summary)
+- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)
 
 ### Project Manager
-OpenRefine's Project Manager (paid position) works closely with OpenRefine’s advisory and steering committees and with its communities of users and contributors.
+OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.
 
 The Project Manager:
-* Helps to improve OpenRefine's governance and community diversity
-* Helps build community-driven structures to formalize OpenRefine’s roadmap and keep it up to date
-* Assists in finding and secure new sources of funding
-* Supports the Advisory Committee in OpenRefine’s day-to-day operations.
+- Helps to improve OpenRefine's governance and community diversity
+- Helps build community-driven structures to formalize OpenRefine's roadmap and keep it up-to-date
+- Assists in finding and securing new sources of funding
+- Supports the Advisory Committee in OpenRefine's day-to-day operations; this may include supervising paid project staff.
 
-The current Project Manager is [Martin Magdinier](https://forum.openrefine.org/u/Martin/summary)
-## Fiscal sponsorship: Code For Science and Society
+The current Project Manager is Martin Magdinier [@magdmartin](https://github.com/magdmartin)and on the [forum](https://forum.openrefine.org/u/Martin/summary).
 
-Since January 2020, OpenRefine is a member project of [Code For Science and Society](https://codeforscience.org/) (CS&S).
+## Member Election
```

Comment:
Is this the New Member Election section referred to above in several places? If so, it should probably be named that.

Reply:
@Ainali, thanks for spotting this. I adjusted how we reference the section through the document